CoreWebServer/src/au/gov/asd/tac/constellation/webserver/services/NewGraph.java

```diff
@@ -113,13 +116,23 @@ public void callService(final PluginParameters parameters, final InputStream in,
         final String graphName = SchemaFactoryUtilities.getSchemaFactory(schemaName).getLabel().trim().toLowerCase();
         GraphOpener.getDefault().openGraph(dualGraph, graphName);
 
-        final String newId = RestServiceUtilities.waitForGraphChange(existingId);
+        String newId = "";
 
-        final ObjectMapper mapper = new ObjectMapper();
-        final ObjectNode root = mapper.createObjectNode();
-        root.put("id", newId);
-        root.put("name", GraphNode.getGraphNode(newId).getDisplayName());
-        root.put("schema", schemaName);
-        mapper.writeValue(out, root);
+        try {
+            newId = RestServiceUtilities.waitForGraphChange(existingId).get(10, TimeUnit.SECONDS);
+        } catch (final InterruptedException ex) {
```

Comment:
are you able to throw an exception here as well (I assume if we interrupt here we probably don't want to continue execution)? Or if it that isn't the intended execution, perhaps a log entry to indicate the interrupt has occurred?